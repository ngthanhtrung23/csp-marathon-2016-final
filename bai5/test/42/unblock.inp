122234
155534
00XY37
TTXY.7
...ABB
...A..
