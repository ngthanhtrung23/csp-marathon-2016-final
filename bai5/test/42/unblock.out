35
X D 2
0 R 1
T R 1
1 D 4
0 L 1
2 L 1
5 L 1
T L 1
X U 2
Y U 2
A U 2
B L 3
3 D 3
7 D 2
4 D 2
A D 2
Y D 2
2 R 3
5 R 3
X U 2
0 R 1
T R 1
1 U 4
0 L 1
B L 1
T L 1
X D 4
0 R 1
T R 1
1 D 2
2 L 3
5 L 3
4 U 2
Y U 2
0 R 3

